src/thunderbots/software/util/parameter/dynamic_parameters.cpp

```diff
@@ -36,6 +36,36 @@ namespace Util::DynamicParameters
                                                          2.0);
     }  // namespace Navigator
 
+    namespace Indirect_Chip_Evaluation
```

Comment:
We're probably going to have more parameters for evaluation, so I think it would make sense to put this one more level down, ie.
```
... 
namespace Evaluation {
    namespace IndirectChip {
...
```

Reply:
Edited as per suggested.

Previously I had the comments for the parameters all in the `.cpp` file because it was done so for the navigator stuff. Decided to migrate them over to the `.h` file which seems to make more sense!

Reply:
Ah, yes, we revamped how parameters are generated to avoid people having to change things in 3 different places every time they wanted to change a Param. I've asked Akhil to post in the #software channel with the details, but basically, if you merge `master` to your branch, you will see a file `software/util/parameter/config/ai.yaml`, where all parameters should go. Then whenever you run `catkin_make`, all the files for parameters (the `.cpp`, `.h`, `.cfg`) will be automagically generated! There should be some examples in there, but let me know if you need any help with this migration.

Reply:
Moved the parameters to `evaluation.yaml` and let it do its automagickery!